gaurun/worker.go

```diff
@@ -51,6 +60,9 @@ Retry:
 
 func pushAsync(pusher func(req RequestGaurunNotification) error, req RequestGaurunNotification, retryMax int, pusherCount *int64) {
 Retry:
+	PusherWg.Add(1)
+	defer PusherWg.Done()
+
```

Comment:
This code is wrong. As **pushAsync()** is called asynchronously, These functions should be called before calling **pushAsync()**.

## FYI

The comment below help you.

https://github.com/mercari/gaurun/blob/feature/hot-deploying/gaurun/worker.go#L108